crates/apollo-mcp-server/src/server/states/starting.rs

```diff
@@ -222,6 +222,7 @@ impl Starting {
                                 "mcp_server",
                                 method = %request.method(),
                                 uri = %request.uri(),
+                                session_id = %request.headers().get("mcp-session-id").and_then(|v| v.to_str().ok()).unwrap_or("none"),
```

Comment:
Have you considered setting it conditionally? Omitting it would be clearer than setting it to `none`.